docs/src/man/joins.md

```diff
@@ -50,3 +50,18 @@ Cross joins are the only kind of join that does not use a key:
 ```julia
 join(a, b, kind = :cross)
 ```
+
+In order to join data frames on keys which have different names, you must first rename them so that they match. This can be done using rename!:
+
+
+```
+rename!(df3, :old_name, :new_name)
```

Comment:
Could you give a real example, i.e. create a data frame `c` with a different column name, rename the column and then join? Would also be nice to create two columns so that you can make the example below work too.